.github/workflows/fetch-and-ingest-gisaid-master.yml

```diff
@@ -3,8 +3,7 @@ name: GISAID fetch and ingest
 on:
   schedule:
     # * is a special character in YAML so you have to quote this string
-    # "At 03:00 (05:00 CET) on every day-of-week from Monday through Friday."
-    - cron:  '0 3 * * MON-FRI'
+    - cron:  '7 13 * * *'
```

Comment:
This means 1:07pm daily to be exact. There's different forms of CRON syntax, but you can use an online tool to decipher the one used by GitHub, from the [official docs](https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#schedule):

> You can use [crontab guru](https://crontab.guru/) to help generate your cron syntax and confirm what time it will run.

Reply:
Yes, I got rid of the MON-SUN since `*` is simpler

I start at 1.07pm because Github doesn't like if you start it on the hour since that's what most people use and it gets crowded.